event.go

```diff
@@ -106,10 +106,10 @@ func GetSystemRenderContext() (SysRenderContext, error) {
 // Get a handle for a event log subscription on the given channel.
 // `query` is an XPath expression to filter the events on the channel - "*" allows all events.
 // The resulting handle must be closed with CloseEventHandle.
-func CreateListener(channel, query string, startpos EVT_SUBSCRIBE_FLAGS, watcher *LogEventCallbackWrapper) (ListenerHandle, error) {
+func CreateListener(channel, query string, startpos EVT_SUBSCRIBE_FLAGS, watcher uint32) (ListenerHandle, error) {
```

Comment:
`watcher` -- rename all occurrences with modified logic to `watcherKey`, `callbackKey` and so on